src/proto/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     list(APPEND CMAKE_PREFIX_PATH "${Protobuf_PREFIX_PATH}")
 endif()
 find_package(Protobuf REQUIRED)
+if (NOT "${Protobuf_VERSION}" VERSION_LESS 4.22)
```

Comment:
Can you add a comment here, something like "protobuf library V4.22+ requires to link absl" to help the understanding of this dependency.

Reply:
Thank you for comments, updated the PR according to the comments.